spec/lib/dfe/reference_data/degrees/combined_subjects_spec.rb

```diff
@@ -35,4 +35,10 @@
       end
     end
   end
+
+  describe 'combined subjects' do
+    it 'is a valid autocomplete-capable list' do
+      DfE::ReferenceData::Degrees::COMBINED_SUBJECTS.validate_autocomplete_compatibility!
+    end
```

Comment:
Great work!
I think we can use a shared example for all the specs.

Something like:

```ruby
RSpec.shared_examples 'a valid autocomplete-capable list' do
  it 'is a valid autocomplete-capable list' do   
    described_class.validate_autocomplete_compatibility!
  end
end
```

And then on other specs you can do:

```ruby
RSpec.describe DfE::ReferenceData::Degrees::GRADES do
  it_should_behave_like 'a valid autocomplete-capable list'
end
```